README.md

```diff
@@ -62,10 +71,40 @@ Decidim::Kids.configure do |config|
     3
   end
 
-  # Other, more advanced, configuration options are available. Check the source code at lib/decidim/kids.rb for more information.
+  # If true, the tutor can impersonate a minor
+  config_accessor :allow_impersonation do
+    true
+  end
+
+  # Default authorization metadata attributes where the minor's birthday is stored
+  # (if the authorization handler stores it)
+  # If this value is present: In addition to the normal verification process for the minor, the
+  #                           age of the minor returned by the validation will be enforced to be
+  #                           between the minimum_minor_age and maximum_minor_age values.
+  #                           Note that if the validation does not stores the birthday in one of these
+  #                           attributes, the validation will always fail.
+  # If this value is blank: No age checks will be performed (but the validation process might do it independently)
+  config_accessor :minor_authorization_age_attributes do
+    [:birthday, :date_of_birth, :birth_date, :birthdate]
+  end
+  
+  # Other, more advanced, configuration options are available.
+  # Check the source code at lib/decidim/kids.rb for more information.
 end
 ```
 
+## Promoting minor accounts to normal accounts
```

Comment:
This section is repeated in the `ARCHITECTURE.md` file mentioned below. We could remove one of the sections

Reply:
ok, remove it.